requirements/constraints_py3.10_pydantic_1.txt

```diff
@@ -181,7 +181,7 @@ npe2==0.7.7
     # via
     #   -r requirements/version_denylist.txt
     #   napari
-numpy==1.26.4
```

Comment:
**suggestion (documentation):** Major version update for numpy

This is a significant version jump from 1.x to 2.x. Please ensure compatibility with your codebase and other dependencies.